src/components/EpisodesControls/EpisodesControls.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useContext } from "react";
+import { data } from "COMPONENTS/Episodes/";
+import { RENDER } from "./";
+import { Episodes } from "COMPONENTS/Episodes/";
```

Comment:
this should be imported from line 2 as you're already importing something from that component above...

Reply:
Done, however Episodes has been renamed in the context branch, so i'll want to update that. Sent you a PM about this as unsure the best way to sync that rename from context.

Reply:
Sorted.